osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -1517,6 +1517,36 @@
       },
       "coingecko_id": "dai"
     },
+    {
+      "description": "Gravity Bridge BTC",
+      "denom_units": [
+        {
+          "denom": "ibc/C9B0D48FD2C5B91135F118FF2484551888966590D7BDC20F6A87308DBA670796",
+          "exponent": 0,
+          "aliases": [
+            "satoshi"
+          ]
+        },
+        {
+          "denom": "btc",
+          "exponent": 8
+        }
+      ],
+      "base": "ibc/C9B0D48FD2C5B91135F118FF2484551888966590D7BDC20F6A87308DBA670796",
+      "name": "Wrapped Bitcoin",
+      "display": "Wrapped BTC",
```

Comment:
'display' property is defining which denomination to show. Please change to 'btc'
`"display": "btc"`

Reply:
Fixed!